src/iconsManifest/supportedExtensions.ts

```diff
@@ -1300,6 +1300,21 @@ export const extensions: IFileCollection = {
       light: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'denoify',
+      extensions: [
+        '.denoifyrc',
+        '.denoifyrc.json',
+        '.denoifyrc.yaml',
+        '.denoifyrc.yml',
+        '.denoifyrc.js',
+        '.denoifyrc.cjs',
+        'denoify.config.js',
+        'denoify.config.cjs',
+      ],
```

Comment:
This should make your code much eaiser to read:

```suggestion
      extensions: ['.denoifyrc'],
      filenamesGlob: ['.denoifyrc', 'denoify.config'],
      extensionsGlob: ['json', 'yaml', 'yml', 'js', 'cjs'],
```

Reply:
I think that this icon can be used for both dark/light theme, if by creating light icon u mean to create one for light theme then yes, I will do it, otherwise, I think it's fine